config/src/lib.rs

```diff
@@ -901,6 +902,53 @@ impl Config {
 
         Ok(())
     }
+
+    /// List the data in the foundry cache
+    pub fn list_foundry_cache() -> eyre::Result<Cache> {
+        if let Some(cache_dir) = Config::foundry_cache_dir() {
+            let mut cache = Cache { chains: vec![] };
```

Comment:
let's simply abort here if `!cache_dir.exists()` ?

Reply:
That's cleaner :+1: thanks for the review